src/aro/Driver.zig

```diff
@@ -47,6 +47,19 @@ color: ?bool = null,
 nobuiltininc: bool = false,
 nostdinc: bool = false,
 nostdlibinc: bool = false,
+debug_dump_letters: packed struct(u3) {
+    d: bool = false,
+    m: bool = false,
+    n: bool = false,
+
+    /// Specifying letters whose behavior conflicts is undefined.
```

Comment:
This could specify that we match clang's behavior.

Reply:
Does it actually match clang as it is now? I just chose the current order because it was alphabetical.

Reply:
Ok just did a quick check - clang doesn't support `-dN`; and `-dM` wins over `-dD` (it always does `-dM` regardless of order if `-dD` and `-dM` are provided). I'll update it to match that.